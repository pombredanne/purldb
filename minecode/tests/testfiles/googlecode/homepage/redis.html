

<!DOCTYPE html>
<html>
<head>
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" >
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" >
 
 <meta name="ROBOTS" content="NOARCHIVE">
 
 <link rel="icon" type="image/vnd.microsoft.icon" href="http://www.gstatic.com/codesite/ph/images/phosting.ico">
 
 
 <script type="text/javascript">
 
 
 
 
 var codesite_token = null;
 
 
 var CS_env = {"domainName": null, "projectName": "redis", "profileUrl": null, "projectHomeUrl": "/p/redis", "loggedInUserEmail": null, "token": null, "relativeBaseUrl": "", "assetHostPath": "http://www.gstatic.com/codesite/ph", "assetVersionPath": "http://www.gstatic.com/codesite/ph/11512667522589661435"};
 var _gaq = _gaq || [];
 _gaq.push(
 ['siteTracker._setAccount', 'UA-18071-1'],
 ['siteTracker._trackPageview']);
 
 _gaq.push(
 ['projectTracker._setAccount', 'UA-174336-10'],
 ['projectTracker._trackPageview']);
 
 (function() {
 var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
 ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
 (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(ga);
 })();
 
 </script>
 
 
 <title>
 redis -
 
 
 A persistent key-value database with built-in net interface written in ANSI-C for Posix systems - Google Project Hosting
 </title>
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/core.css">
 
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/ph_detail.css" >
 
 
 
 
<!--[if IE]>
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/d_ie.css" >
<![endif]-->
 <style type="text/css">
 .menuIcon.off { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -42px }
 .menuIcon.on { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -28px }
 .menuIcon.down { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 0; }
 
 
 .psicon-container {
 min-width:24px;
 }

 
 </style>
</head>
<body class="t1">
<script type="text/javascript">
 window.___gcfg = {lang: 'en'};
 (function() 
 {var po = document.createElement("script");
 po.type = "text/javascript"; po.async = true;po.src = "https://apis.google.com/js/plusone.js";
 var s = document.getElementsByTagName("script")[0];
 s.parentNode.insertBefore(po, s);
 })();
</script>
<div class="headbg">

 <div id="gaia">
 

 <span>
 
 
 <a href="#" id="projects-dropdown" onclick="return false;"><u>My favorites</u> <small>&#9660;</small></a>
 | <a href="https://www.google.com/accounts/ServiceLogin?service=code&amp;ltmpl=phosting&amp;continue=http%3A%2F%2Fcode.google.com%2Fp%2Fredis%2F&amp;followup=http%3A%2F%2Fcode.google.com%2Fp%2Fredis%2F" onclick="_CS_click('/gb/ph/signin');"><u>Sign in</u></a>
 
 </span>

 </div>

 <div class="gbh" style="left: 0pt;"></div>
 <div class="gbh" style="right: 0pt;"></div>
 
 
 <div style="height: 1px"></div>
<!--[if lte IE 7]>
<div style="text-align:center;">
Your version of Internet Explorer is not supported. Try a browser that
contributes to open source, such as <a href="http://www.firefox.com">Firefox</a>,
<a href="http://www.google.com/chrome">Google Chrome</a>, or
<a href="http://code.google.com/chrome/chromeframe/">Google Chrome Frame</a>.
</div>
<![endif]-->



 <table style="padding:0px; margin: 0px 0px 10px 0px; width:100%" cellpadding="0" cellspacing="0"
 itemscope itemtype="http://schema.org/CreativeWork">
 <tr style="height: 58px;">
 
 
 
 <td id="plogo">
 <link itemprop="url" href="/p/redis">
 <a href="/p/redis/">
 
 
 <img src="/p/redis/logo?cct=1316166100"
 alt="Logo" itemprop="image">
 
 </a>
 </td>
 
 <td style="padding-left: 0.5em">
 
 <div id="pname">
 <a href="/p/redis/"><span itemprop="name">redis</span></a>
 </div>
 
 <div id="psum">
 <a id="project_summary_link"
 href="/p/redis/"><span itemprop="description">A persistent key-value database with built-in net interface written in ANSI-C for Posix systems</span></a>
 
 </div>
 
 
 </td>
 <td style="white-space:nowrap;text-align:right; vertical-align:bottom;">
 
 <form action="/hosting/search">
 <input size="30" name="q" value="" type="text">
 
 <input type="submit" name="projectsearch" value="Search projects" >
 </form>
 
 </tr>
 </table>

</div>

 
<div id="mt" class="gtb"> 
 <a href="/p/redis/" class="tab active">Project&nbsp;Home</a>
 
 
 
 
 <a href="/p/redis/downloads/list" class="tab ">Downloads</a>
 
 
 
 
 
 <a href="/p/redis/wiki/index?tm=6" class="tab ">Wiki</a>
 
 
 
 
 
 <a href="/p/redis/issues/list"
 class="tab ">Issues</a>
 
 
 
 
 
 <a href="/p/redis/wiki/UnstableSource?tm=4"
 class="tab ">Source</a>
 
 
 
 
 
 
 
 
 <div class=gtbc></div>
</div>
<table cellspacing="0" cellpadding="0" width="100%" align="center" border="0" class="st">
 <tr>
 
 
 
 
 
 
 
 <td class="subt">
 <div class="st1">
 <div class="isf">
 <span class="inst1">
 <a href="/p/redis/">Summary</a>
 </span>
 
 
 &nbsp;
 <span class="inst3">
 <a href="/p/redis/people/list">People</a>
 </span>
 
 
 </div>
</div>

 </td>
 
 
 <td align="right" valign="top" class="bevel-right"></td>
 </tr>
</table>


<script type="text/javascript">
 var cancelBubble = false;
 function _go(url) { document.location = url; }
</script>
<div id="maincol"
 
>

 





<table width="100%">
 <tr class="pscontent">
 <td class="pscolumnl">
 
 <div class="phead">Project Information</div>
 
 
 
 <ul class="pslist">
 
 <li class="psmeta">
 <a href="/p/redis/feeds">Project feeds</a>
 </li>
 <li class="psgap"/>
 
 
 
 <li class="psmeta"><b>Code license</b></li>
 <li class="psmeta">
 <a href="http://opensource.org/licenses/BSD-3-Clause" rel="nofollow">New BSD License</a>
 
 </li>
 <li class="psgap"/>
 
 
 
 
 
 
 <li class="psmeta">
 <span id="project_labels">
 <b>Labels</b><br/>
 
 <a class="label"
 href="/hosting/search?q=label:redis">redis</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:KeyValue">KeyValue</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:AnsiC">AnsiC</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:database">database</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:networking">networking</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:persistent">persistent</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:lists">lists</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:sets">sets</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:atomic">atomic</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:memcached">memcached</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:Posix">Posix</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:GPL">GPL</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:dictionary">dictionary</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:hashTable">hashTable</a>
 
 </span>
 </li>
 <li class="psgap"/>
 
 
 </ul>
 <div class="psicon">
 <div class="psicon-container goog-inline-block">
 <div style="float:right" class="SPRITE_people-y16 goog-inline-block vt"></div>
 </div>
 <span><b>Members</b></span>
 </div>
 <ul class="pslist">
 
 
 
 

 <a class="userlink"
 href="/u/116085733329177489569/">anti...@gmail.com</a
>
 
 
 
 
 <li class="psmeta">
 <a href="people/list">9 contributors</a>
 </li>
 
 <li class="psgap"/>
 
 
 </ul>
 
 
 
 
 <div class="phead">Links</div>
 <ul class="pslist" style="white-space:nowrap">
 
 <li class="psmeta"><b>Blogs</b></li>
 
 <li class="psmeta">
 <a href="http://twitter.com/antirez" rel="nofollow">Twitter updates</a>
 </li>
 
 <li class="psgap"/>
 
 
 
 <li class="psmeta"><b>Groups</b></li>
 
 <li class="psmeta">
 <a href="http://groups.google.com/group/redis-db" rel="nofollow">Redis DB dev and usage</a>
 </li>
 
 <li class="psgap"/>
 
 </ul>
 
 </td>
 <td id="wikicontent" class="psdescription">
 <h1><a name="We_moved_to_Redis.io!"></a>We moved to Redis.io!<a href="#We_moved_to_Redis.io!" class="section_anchor"></a></h1><p>Hello! The Redis project home page moved to <a href="http://redis.io" rel="nofollow">redis.io</a>. </p>
 </td>
 </tr>
</table>
<script src="http://www.gstatic.com/codesite/ph/11512667522589661435/js/prettify/prettify_core_compiled.js"></script>
<script type="text/javascript">
 prettyPrint();
</script>

 
 
 
 <script type="text/javascript" src="http://www.gstatic.com/codesite/ph/11512667522589661435/js/ph_core.js"></script>
 
 
 
 
</div> 

<div id="footer" dir="ltr">
 <div class="text">
 <a href="/projecthosting/terms.html">Terms</a> -
 <a href="http://www.google.com/privacy.html">Privacy</a> -
 <a href="/p/support/">Project Hosting Help</a>
 </div>
</div>
 <div class="hostedBy" style="margin-top: -20px;">
 <span style="vertical-align: top;">Powered by <a href="http://code.google.com/projecthosting/">Google Project Hosting</a></span>
 </div>

 
 


 
 </body>
</html>

