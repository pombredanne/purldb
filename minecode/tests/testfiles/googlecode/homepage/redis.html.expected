{'content_license': None,
 'content_license_url': None,
 'description': u'A persistent key-value database with built-in net interface written in ANSI-C for Posix systems',
 'downloads_url': u'http://code.google.com/p/redis/downloads/list',
 'external_links': {},
 'feeds_url': u'http://code.google.com/p/redis/feeds',
 'home_url': u'http://code.google.com/p/redis/',
 'id': u'redis',
 'issues_url': u'http://code.google.com/p/redis/issues/list',
 'keywords': [],
 'labels': [u'AnsiC',
            u'GPL',
            u'KeyValue',
            u'Posix',
            u'atomic',
            u'database',
            u'dictionary',
            u'hashTable',
            u'lists',
            u'memcached',
            u'networking',
            u'persistent',
            u'redis',
            u'sets'],
 'license': u'New BSD License',
 'license_url': u'http://opensource.org/licenses/BSD-3-Clause',
 'name': u'redis',
 'peoples_url': u'http://code.google.com/p/redis/people/list',
 'vcs_url': u'http://code.google.com/p/redis/wiki/UnstableSource?tm=4',
 'wiki_url': u'http://code.google.com/p/redis/wiki/index?tm=6'}